File: trains.client/src/app/models/stopTime.ts
import { Time } from "@angular/common";
import { IRoad } from "./road";
import { IStation } from "./station";


export interface IStopTime {
  Id: number;
  RoadId: number;
  Road: IRoad;
  StationId: number;
  Station: IStation;
  Time: Time;
}

File: Trains.Server/Data/Models/StopTime.cs
namespace Trains.Server.Data.Models
{
    public class StopTime
    {
        public int Id { get; set; }
        public int RoadId { get; set; }
        public Road Road { get; set; }
        public int StationId { get; set; }
        public Station Station { get; set; }
        public TimeSpan Time { get; set; }
    }
}
